guacamole-server.yaml

```diff
@@ -56,30 +56,38 @@ pipeline:
 
   - uses: strip
 
+  - runs: |
+      find ${{targets.destdir}}/usr/lib -name '*.a' -type f -exec rm -f '{}' +
+
+data:
+  - name: plugins
+    items:
+      vnc: VNC
+      telnet: Telnet
```

Comment:
maybe we can add `ssh` here as the @dlorenc did in his PR: https://github.com/wolfi-dev/os/pull/3243/files

Reply:
then I can add rdp here as well